packages/samples/react/src/scenarios/appointment-form/DistrictForm.tsx

```diff
@@ -27,25 +27,30 @@ const LOCATION_OPTIONS = [
 	},
 ];
 
+type ErrorListPropType = {
+	message: string;
+	selector: string;
+};
+
 export function DistrictForm() {
 	const form = useFormikContext<FormValues>();
-	const [sectionSubmitted, setSectionSubmitted] = useState(false);
+	const errorList = createErrorList(form.errors);
+
+	function createErrorList(formikErros: Record<string, string>): ErrorListPropType[] {
```

Comment:
```suggestion
	function createErrorList(formikErrors: Record<string, string>): ErrorListPropType[] {
```
Kleiner typo, bitte einmal überall suchen & ersetzen :)